ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-release-4.7.yaml

```diff
@@ -3,10 +3,10 @@ base_images:
     name: "4.7"
     namespace: ocp
     tag: base
-  ocp_builder_rhel-7-golang-1.13:
+  ocp_builder_rhel-7-golang-1.14:
```

Comment:
this isn't even being used, you should just remove it.

same for `ocp_builder_rhel-8-golang-openshift-4.6` and `ocp_builder_rhel-8-golang-1.15-openshift-4.6`

`base` and `ocp_builder_rhel-8-golang-1.15-openshift-4.7` are the only images this CI config is consuming.

Reply:
updated in current push